src/docs/cursor-ops.dox

```diff
@@ -161,4 +161,52 @@ that may not be modified or freed by the application.  If a longer scope
 is required, the application must make a copy of the memory before the
 cursor is re-used, closed or reset.
 
+@section cursor_bound Apply bounds to the cursor
+
+A cursor may have a set of bounds applied to it, this restricts the key range
+that the cursor operates within.
+
+Suppose we have a table with keys A to Z. We can restrict the cursor to only operate over
+keys L->Q. If the cursor walks past key Q, the cursor will enter an un-positioned state having
+walked off the "end" of the table from it's perspective. Additionally should the cursor perform a
```

Comment:
`it's` -> `its`

Reply:
[9b21f18](https://github.com/wiredtiger/wiredtiger/pull/8367/commits/9b21f186bb973ba5c89e96925bf8becfecf64b56)